cfme/infrastructure/cluster.py

```diff
@@ -137,6 +148,15 @@ def am_i_here(self):
         return match_page(summary="{} (Summary)".format(self.obj.name))
 
 
+@navigator.register(Cluster, 'Timelines')
+class Timelines(CFMENavigateStep):
```

Comment:
I don't believe this works as expected. If you navigate to `Details`, and want to move to `Timelines` from there, it won't navigate any further so the check seems incorrect.